README.md

```diff
@@ -28,6 +28,18 @@ Run npm install in fact-bounty-client folder.
  cd fact-bounty-client/
  npm install
  ```
+* Add following variables to react app .env file
+
+```
+        REACT_APP_GOOGLE_CLIENT_ID=""
+        REACT_APP_FACEBOOK_CLIENT_ID=""
+        REACT_APP_OAUTHD_KEY=""
+        REACT_APP_OAUTHD_URL=""
+```
+    Save the file.
+
+    This variables can be also found in env.example file in react app.
```

Comment:
@Binurim let's change this content. Since there is an env.example file already, mention here to rename this file as .env. And then also mention to set up these values in order to use the twitter search function and google, facebook login if possible.

Reply:
@root5533 Changes are done and squashed all commits. Can you please review this.